### ISSUE TITLE
Bot is prevent issues from being stale

### ISSUE THREAD

**Issue by prince-chrismc** [ericLemanissier/conan-center-conflicting-prs#7]
Take https://github.com/conan-io/conan-center-index/pull/3367 for instance from October which was not marked stale from the bot's comment 23 days ago.

It would be nice to **not** comment on older PRs sp they can become stale

**Comment by ericLemanissier** [comment]
Thanks for the report. I think I will prevent the bot from posting messages on PR which had no activity for the last 2 weeks. do you agree ?

**Comment by prince-chrismc** [comment]
Yes that would be amazing!

**Comment by prince-chrismc** [comment]
Still seems to be an issue, https://github.com/conan-io/conan-center-index/pull/2962#issuecomment-742786281 removed the stale

**Comment by ericLemanissier** [comment]
Right, checking the "updated on" field is not enough, because the stale bot updates prs by posting a message. I'll add another check on "stale" label tomorrow.

**Comment by prince-chrismc** [comment]
Thanks!

I can wait for the 40 stale PRs to be gone 😃

**Comment by prince-chrismc** [comment]
The problem thickens! 

The stale bot removed the tag in https://github.com/conan-io/conan-center-index/pull/1704

Pulling up the API https://api.github.com/repos/conan-io/conan-center-index/pulls/1704

>   `"updated_at": "2020-12-24T06:18:40Z",`

Which is the time the PR was last mentioned....

**Comment by ericLemanissier** [comment]
Stupid mistake, it should be fixed now, sorry :(

**Comment by prince-chrismc** [comment]
No worries, mines going to have the same problem!

**Comment by Croydon** [comment]
If I understand these changes correctly, it would still mean, that if there is a single NEW PR for e.g. Boost every < 14 days, the bot still keeps ALL Boost PRs from getting stale.

The only way to truly prevent that is probably to get the date of the last activity which was not this bot itself. Please correct me if I am missing something

**Comment by ericLemanissier** [comment]
Indeed, I'll work on this in the coming days.
I checked all the PRs detected by this bot, on the only one affected by this is https://github.com/conan-io/conan-center-index/pull/3918
